UnitsNet.Tests/CustomCode/CompressibilityTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by \generate-code.bat.
+//
+//     Changes to this file will be lost when the code is regenerated.
+//     The build server regenerates the code before each build and a pre-build
+//     step will regenerate the code on each local build.
+//
+//     See https://github.com/angularsen/UnitsNet/wiki/Adding-a-New-Unit for how to add or edit units.
+//
+//     Add CustomCode\Quantities\MyQuantity.extra.cs files to add code to generated quantities.
+//     Add UnitDefinitions\MyQuantity.json and run generate-code.bat to generate new units or quantities.
+//
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+// Licensed under MIT No Attribution, see LICENSE file at the root.
+// Copyright 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com). Maintained at https://github.com/angularsen/UnitsNet.
+
+using System;
+
+namespace UnitsNet.Tests.CustomCode
+{
+    public class CompressibilityTests : CompressibilityTestsBase
+    {
+        // Override properties in base class here
+
+        protected override double InversePascalsInOneInversePascal => 1;
+        protected override double InverseMillibarsInOneInversePascal => 1e-2;
+        protected override double InverseKilopascalsInOneInversePascal => 1e-3;
+        protected override double InverseMegapascalsInOneInversePascal => 1e-6;
+        protected override double InverseBarsInOneInversePascal => 1e-5;
+        protected override double InversePoundForcePerSquareInchsInOneInversePascal => 1.450377377302092151542e-4;
```

Comment:
Typo `Inchs`

Reply:
> Typo `Inchs`

Ah indeed. It should be ```InversePoundsForcePerSquareInch``` anyways. Fixed!